lib/Libki/Controller/Administration/Settings.pm

```diff
@@ -34,12 +34,45 @@ sub index :Path :Args(0) {
     my ( $self, $c ) = @_;
 
     my $instance = $c->instance;
-
+    my @timeallowance;
+    my @timesession;
     my $settings = $c->model('DB::Setting')->search({ instance => $instance });
-    
+    my @locations = $c->model('DB::Client')->search(
```

Comment:
We have a locations table, so we should probably search the locations table in a followup patch.